CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ set(APFP_TILE_SIZE_M 32 CACHE STRING "Tile size in the M-dimension when running
 set(APFP_SEMANTICS "MPFR" CACHE STRING "Which semantics to use for floating point operations [GMP/MPFR].")
 set(APFP_PROFILING OFF CACHE BOOL "Enable profiling in the generated kernel.")
 set_property(CACHE APFP_SEMANTICS PROPERTY STRINGS GMP MPFR)
+set(APFP_INTERFACE_TYPE "MPFR" CACHE STRING "Which data types to use for the interface [GMP/MPFR].")
```

Comment:
Right now the idea is that we can stick with the MPFR kernel that we know is working while having the interface library handle the GMP <-> MPFR conversion. With SDPB I am using the GMP interface and using the MPFR kernel, since that is the one that is passing the tests

Reply:
Converting between GMP and MPFR representations is expensive, do we really want to do that on the fly? I would probably just use SDPB in MPFR mode for now to make our lives easier, if that's an option

Reply:
Aren't we converting to/from PackedFloat anyway? I was under the impression the MPFR vs GMP flag on the kernel side just changed some rounding, etc? I'm not concerned about exact bit-by-bit reproducibility because afaik SDPB doesn't rely on exact rounding rules, etc

It looks like MPFR vs GMP support in SDPB is a fairly small patch so we could go that route if it results in different performance

Reply:
I just took a look at the GMP/MPFR <-> PackedFloat code and it seems like we shouldn't be getting any perf hit beyond converting to/from PackedFloat?

Reply:
Okay, that was definitely not what I was thinking the behavior was. In that case we should merge it yeah